include/TGUI/Widgets/ChildWindow.hpp

```diff
@@ -199,6 +214,43 @@ namespace tgui
 
 
         /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+        /// @brief Changes the title buttons.
+        ///
+        /// @param buttons  Which buttons should be available in the title bar?
+        ///
+        /// By default ChildWindows only display a close button. You may set the window to show a combination of buttons.
+        /// For example, the following will set the ChildWindow to have both a minimize and close button.
+        /// @code
+        /// childWindow->setTitleButtons(tgui::ChildWindow::TitleButtons::Minimize | tgui::ChildWindow::TitleButtons::Close);
```

Comment:
That's probably the worst excuse I have ever made: I did a quick lookup and found only 8 lines that use "tgui::" as part of a parameter in the documentation :smile:. The other lines are places where I use e.g. "tgui::Picture::Ptr" as type instead of "Picture::Ptr". And the one in ChildWindow is definately the longest line.